plugins/Monolog/Handler/SystemLogHandler.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\Monolog\Handler;
+
+use Monolog\Handler\SyslogHandler;
+
+/**
+ * A class to use the SyslogHandler instead of including the dependency directly
+ */
+
+class SystemLogHandler extends SyslogHandler
```

Comment:
Would it be possible to keep the name? Or is it better to also rename the class when it's not changing any behaviour and is simply a wrapper?

```suggestion
use Monolog\Handler\SyslogHandler as MonologSyslogHandler;

/**
 * A class to use the SyslogHandler instead of including the dependency directly
 */

class SyslogHandler extends MonologSyslogHandler
```

No strong feelings about it though, just a thought. What do you think @mneudert?

Reply:
Keeping the name sounds like a good idea. A "system log" could mean something different for someone that is used to the `syslog` naming (and it being tied to the syslog program) :thinking: